docs/source/guide.rst

```diff
@@ -150,56 +150,10 @@ actor an invalid URL.  Let's try it::
   >>> count_words.send("foo")
 
 
-Error Handling
---------------
-
-Remoulade assumes all actors are idempotent so when an exception occurs
-during message processing, it automatically enqueues a retry for that
-message with exponential backoff.
-
-That last message we sent will cause something along these lines to be
-printed in your worker process::
-
-  [2017-06-27 13:11:22,059] [PID 13053] [Thread-8] [remoulade.worker.WorkerThread] [WARNING] Failed to process message count_words('foo') with unhandled exception.
-  Traceback (most recent call last):
-    ...
-  requests.exceptions.MissingSchema: Invalid URL 'foo': No schema supplied. Perhaps you meant http://foo?
-  [2017-06-27 13:11:22,062] [PID 13053] [Thread-8] [remoulade.middleware.retries.Retries] [INFO] Retrying message 'a53a5a7d-74e1-48ae-a5a8-0b72af2a8708' in 8104 milliseconds.
-
-Remoulade will keep retrying the message with longer and longer delays
-in between runs until we fix our code or for up to about 30 days from
-when it was first enqueued.
-
-Change ``count_words`` to catch the missing schema error::
-
-   @remoulade.actor
-   def count_words(url):
-     try:
-       response = requests.get(url)
-       count = len(response.text.split(" "))
-       print(f"There are {count} words at {url!r}.")
-     except requests.exceptions.MissingSchema:
-       print(f"Message dropped due to invalid url: {url!r}")
-
-Then send ``SIGHUP`` to the main worker process to make the workers
-pick up the source code changes::
-
-  $ kill -s HUP 13047
-
-Substitute the process ID of your own main process for ``13047``.  You
-can find the PID by looking at the log lines from the worker starting
-up.  Look for lines containing the string ``[remoulade.MainProcess]``.
-
-The next time your message is retried you should see::
-
-  Message dropped due to invalid url: 'foo'
-
-
 Message Retries
 ---------------
 
-As mentioned in the error handling section, Remoulade automatically
-retries failures with exponential backoff.
+Alternatively, you can add the |Retries| Middleware to the broker and set the max_retries or retry_when option to automatically retry your message on failure.
```

Comment:
Add a sentence before like `If an error occurs during the processing of a message, it will be terminated with a failure message.